README.md

```diff
@@ -198,6 +199,54 @@ If you are exclusively using Passkeys as a secondary verification step, you don'
 
 9. That's it! You should now see a "Register Passkey" button on your logged-in user template. Clicking this button will start the registration process. After registration, you should see a "Login using a Passkey" button on your login page. Clicking this button will prompt you to use your Passkey to authenticate. Or if your browser supports it, you will be prompted to use your Passkey when you focus the username field.
 
+## Using custom credential and attestation models.
+
+Django OTP WebAuthn provides its own models for credentials and attestations for those credentials. If these do not suit your needs you can also provide your own models. When using a custom credential model them you *must* use a custom attestation model as well.
```

Comment:
**Thought:** we should explain better what attestations do to the uniniated, but that is a topic for the docs which have yet to be written...